applications/luci-app-aria2/root/usr/libexec/aria2-call

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+action=$1
+shift
+
+[ -r "$1" ] || exit 1
+
+case "$action" in
+	cat)
+		cat $1
+	;;
+	tail)
+		tail -n 50 $1
```

Comment:
Instead of allowing arbitrary files here you should pull these values out of the uci configuration.

Example:
```bash
log_file=$(uci -q get aria2.@aria2[0].log)
tail -n 50 "${log_file:-/var/log/aria2.log}"
```

Reply:
Done